apps/etterlatte-behandling/src/test/kotlin/behandling/BehandlingFactoryTest.kt

```diff
@@ -450,7 +450,7 @@ class BehandlingFactoryTest {
 
     @Test
     fun `skal ikke kunne opprette omgjøring førstegangsbehandling hvis det er innvilget førstegangsbehandling`() {
-        val sakId = 1L
+        val sakId = sakId1
```

Comment:
Synes ikke det er veldig god praksis å gjenbruke verdier på tvers av tester. Foretrekker f.eks. `val sakId = Random.nextLong()` ettersom man da er sikker på at ID-en er unik for hver enkelt test og man ikke får grønne tester som _potensielt_ ville feilet ellers... 🤔

Reply:
Einig i det. Vurderer å ta ein runde etter denne endringa er ferdig og bytte ut all bruk av desse tre hardkoda variantane med den tilfeldige, men ville ikkje blande inn det i den allereie store omskrivinga.